utils/dataset_manifest/errors.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2022 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+class Error(Exception):
+    """
+    The basic exception type for all exceptions in the library
+    """
+
+class InvalidVideoFrameError(Error):
+    pass
+
+class InvalidManifestError(Error):
+    pass
+
+class InvalidManifestFieldError(InvalidManifestError):
+    pass
+
+class MissingFieldError(InvalidManifestFieldError):
+    pass
+
+class InvalidFieldValueError(InvalidManifestFieldError):
+    pass
```

Comment:
These fine-grained exception types seem unnecessary. They're not handled any differently from the base type, and this is not a library, so there's no external user who could handle them differently.